.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior with a reproducible whenever possible.
+
+**Expected behavior**
```

Comment:
Also, if there is an "Expected behavior," I think there should also be a "Current behavior" section showing or trying to show the bug.

Reply:
This might be a little duplicated with the description (first section), but I will add anyway.

Reply:
Okay, or maybe in the first section, add something like "describe the problem and add the current behavior" I think that is enough to do not duplicate contents.